product_sales_record/models/product_template.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons import decimal_precision as dp
+
+from odoo import models, fields, api
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    list_price_record = fields.Monetary(
```

Comment:
Although it's rather minor, "list_price_record" sounds a bit strange.  I'd suggest "sale_price_unit" instead.